setup.py

```diff
@@ -305,10 +305,13 @@ def get_basesources(name, osname):
                    py_limited_api = py_limited_api,
                   )
 ext_modules.append(IOPort)
+description = 'Pieces of Psychtoolbox-3 ported to CPython.'
 
 setup (name = 'psychtoolbox',
        version = version,
-       description = 'Pieces of Psychtoolbox-3 ported to CPython.',
+       description = description,
+       long_description = description,
+       long_description_content_type = 'text/x-rst',
```

Comment:
Running a `twine check --strict` it's unhappy if there is no `long_description`, so I set it to the same as the regular description and set the content-type to reST (though I don't think it matters much).